docs/how-to/how-to-run-dind-with-kata.md

```diff
@@ -0,0 +1,122 @@
+# How to run Docker in Docker with Kata Containers
+
+This document describes the why and how behind running Docker-in-Docker ("dind") in a kata container.
+
+## Requirements
+
+- A working Kata Containers installation
+
+## Install and configure Kata Containers
+
+Follow the [Kata Containers installation guide](../docs/install/README.md) to Install Kata Containers on your Kubernetes cluster.
+
+## Background
+
+Docker in Docker ("DinD") is the colloquial name for the ability to run `docker` from inside a container (whether provided by `docker` or otherwise) pattern for running `docker` from *inside* an existing `docker` container. You can read more about Docker-in-docker at the following links:
+
+- [The original announcement of DinD](https://www.docker.com/blog/docker-can-now-run-within-docker/)
+- [`docker` image DockerHub page](https://hub.docker.com/_/docker/) (this page lists the `-dind` releases)
```

Comment:
```suggestion
- [`docker` image Docker Hub page](https://hub.docker.com/_/docker/) (this page lists the `-dind` releases)
```

`DockerHub` is not part of our dictionary, but `Docker Hub`seems to be fine :-)

Reply:
Ah I'll fix this as well -- since I'm rewriting the commit I'll do this manually rather than committing this suggestion

- [ ] `DockerHub` -> `Docker Hub` on line 18